code/modules/antagonists/nightmare/nightmare.dm

```diff
@@ -13,4 +13,5 @@
 	name = "Nightmare (Preview only)"
 
 /datum/outfit/nightmare/post_equip(mob/living/carbon/human/human, visualsOnly)
-	human.set_species(/datum/species/shadow/nightmare)
+	if(SSoverlays.initialized == TRUE)
+		human.set_species(/datum/species/shadow/nightmare) // SKYRAT EDIT: Ours does not work with nightmares, so we have a special copy of the TG version!
```

Comment:
Missing the SKYRAT edits around it, to make it wrapped by the comments, since it's more than one line.